### ISSUE TITLE
Is this repository abandoned?

### ISSUE THREAD

**Issue by fzaninotto** [KyleAMathews/react-headroom#150]
Hi,

Thanks for the great work.

I see no activity (code commit) since about a year. Is this repository retired, or should we expect some bug fixes from time to time? Some issues (like #134) make this library almost unusable as it is.

**Comment by ckeeney** [comment]
I just stumbled across this project for the first time, and [the demo](https://kyleamathews.github.io/react-headroom/) looks like exactly what I want.

I was disappointed to see the latest commit to master is 11 months ago today.  

The fact that someone opened this issue, someone else liked it, and then I stumbled across it all within 11 hours suggests this library still gets a fair amount of traffic/interest.

**Comment by KyleAMathews** [comment]
Not abandoned — just stable. Happy to take a PR for #134 

Code doesn't stop working just cause there's not new commits :-)

**Comment by fzaninotto** [comment]
Nice to see that you're open to contributions. 😄 

But I tried to contribute, and failed. I cloned your repo, ran `yarn`, then `yarn watch`, and got an error. Then I dug into the code, opened the `www` directory, ran `yarn` again, then (how did I know that? It's documented nowhere) ran `./node_modules/.bin/gatsby develop`, and I got an error. Maybe I'm typing the wrong commands, but how should I know? Your repo doesn't explain how to test the code on a sample project.

In short, if you welcome contributions, please make it so people can contribute.

**Comment by KyleAMathews** [comment]
This is a side project 🤷‍♂️ if you'd like to add more contribution instructions based on what you've learned please do!
